.travis.yml

```diff
@@ -33,7 +33,7 @@ before_script:
  - mysql -e "create database IF NOT EXISTS twfy_test;" -uroot
  - mysql -u root twfy_test < db/schema.sql
  - php composer.phar install --no-interaction --prefer-source
- - bash -c 'if [ "$TRAVIS_PHP_VERSION" == "7.0" ]; then echo "extension = memcached.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini; else echo "extension = memcache.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini; fi;'
+ - bash -c 'if [ "$TRAVIS_PHP_VERSION" == "7.0" ] || [ "$TRAVIS_PHP_VERSION" == "7.1" ]; then echo "extension = memcached.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini; else echo "extension = memcache.so" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini; fi;'
```

Comment:
I wonder if this could be a >= but bash so not sure.

Reply:
I think it would be: `if [ "$TRAVIS_PHP_VERSION" -ge "7.0" ]`

Reply:
Turns out bash can't do floating point comparison, but we can push it through `bc` and achieve the same result.